pkg/validatePoliciesYaml/validatePoliciesYaml_test.go

```diff
@@ -53,11 +71,17 @@ func TestValidatePoliciesYaml(t *testing.T) {
 	assertValidationResult(t, missingPolicyRules, "./test_fixtures/missingPolicyRules.yaml", nil)
 	assertValidationResult(t, missingApiVersion, "./test_fixtures/missingApiVersion.yaml", errors.New("found errors in policies file ./test_fixtures/missingApiVersion.yaml:\n(root): missing properties: 'apiVersion'"))
 	assertValidationResult(t, missingPolicyName, "./test_fixtures/missingPolicyName.yaml", errors.New("found errors in policies file ./test_fixtures/missingPolicyName.yaml:\n(root)/policies/0: missing properties: 'name'"))
+	assertValidationResult(t, missingSchema, "./test_fixtures/missingSchema.yaml", errors.New("invalid policy file: rule identifier \"PODDISRUPTIONBUDGET_DENY_ZERO_VOLUNTARY_DISRUPTION\" has no associated schema"))
```

Comment:
missing test case & check for case where the customRule has both `schema` & `jsonSchema` keys, should get the same error message:

"found errors in policies file ./test_fixtures/missingSchema.yaml:\n(root)/customRules/1: Exactly one of [schema,jsonSchema] should be defined per custom rule"

Reply:
Thanks for pointing out :)